fe/fe-core/src/main/java/com/starrocks/alter/SchemaChangeHandler.java

```diff
@@ -1310,6 +1376,90 @@ public void updatePartitionsInMemoryMeta(Database db,
      * Update one specified partition's in-memory property by partition name of table
      * This operation may return partial successfully, with a exception to inform user to retry
      */
+    public void updateBinlogPartitionTabletMeta(Database db,
```

Comment:
Comment and function not matched?

Reply:
How about make this function batch processing multi-partitions to reduce latency?